dev/scripts/watch-metrics.sh

```diff
@@ -98,6 +101,18 @@ if [[ -z "${host}" || -z "${port}" || -z "${metrics_file_prefix}" || -z "${metri
     exit 1
 fi
 
+# check if command `nslookup` exists
+if ! command -v nslookup > /dev/null 2>&1; then
+  echo "ERROR: nslookup command not found"
+  exit 1
+fi
+
+# check if command `jq` exists
+if ! command -v jq > /dev/null 2>&1; then
+  echo "ERROR: jq command not found"
+  exit 1
+fi
+
 nslookup ${host} > /dev/null 2>&1
```

Comment:
`nslookup` does not read `/etc/hosts` for locally configured hostnames. This preemptive check is too restrictive. Let's just remove this check and let `curl` complain when the host name is unknown.

ref: https://serverfault.com/questions/303716/linux-command-line-utility-to-resolve-host-names-using-etc-hosts-first

Reply:
Got it.